pkgs/development/python-modules/quantities/default.nix

```
@@ -12,21 +13,31 @@
 buildPythonPackage rec {
   pname = "quantities";
   version = "0.15.0";
-  format = "setuptools";
+  pyproject = true;
 
   disabled = pythonOlder "3.8";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-nqMeKg11F88k1UaxQUbe+SkmOZk6YWzKYbh173lrSys=";
+  src = fetchFromGitHub {
+    owner = "python-quantities";
+    repo = "python-quantities";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-N20xfzGtM0VnfkJtzMytNLySTkgVz2xf1nEJxlwBSCI=";
   };
 
-  nativeBuildInputs = [
+  patches = [
+    (fetchpatch2 {
+      name = "prevent-arbitrary-code-eval.patch";
+      url = "https://github.com/python-quantities/python-quantities/pull/236.patch";
```

```suggestion
      url = "https://github.com/python-quantities/python-quantities/commit/329433cac3776d58b208952e328014f716cdfa92.patch";
```

Direct commit references are usually preferred (hash unchanged in this case)

---

Since the PR has been merged this is fine.